tests/unittests/test_google_sheets_supportsAllDrives_In_config.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from unittest import mock
+from tap_google_sheets import GoogleClient
+
+class TestSupportsAllDrives(unittest.TestCase):
+    def test_supportsAllDrives_not_in_config_file(self):
+        """To verify that when supportsAllDrives are not given in config.json then set default value True"""
```

Comment:
Default value should be false.

Reply:
I have changed the default value to False, and also written a unit test according to the default value.